Docs/Makefile

```diff
@@ -21,13 +21,6 @@ clean:
 # Catch-all target: route all unknown targets to Sphinx using the new
 # "make mode" option.  $(O) is meant as a shortcut for $(SPHINXOPTS).
 %: Makefile
+	doxygen
 	@$(SPHINXBUILD) -M $@ "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
-
-Python-build:
```

Comment:
Did you manually or automatically modify the Makefile, which is usually auto-generated by a sphinx script?

Reply:
That was manual modifications. I reverted them and it does not change much.